glui_clipboard.cpp

```diff
@@ -0,0 +1,303 @@
+/****************************************************************************
+
+  GLUI User Interface Toolkit
+  ---------------------------
+
+     glui_clipboard_x11.cpp - GLUI_Clipboard_Object class
+
+	 This file -is- in the public domain. Circa summer 2017.
```

Comment:
I don't think it's healthy for a project to complicate the licensing.  Adopters want a simple answer, not a complicated one.